locale/vi/LC_MESSAGES/django.po

```diff
@@ -5404,6 +5404,14 @@ msgstr "Xem thành viên tham gia"
 msgid "Show full name/organization"
 msgstr "Hiển thị tên/tổ chức"
 
+#: templates/contest/ranking.html:579
+msgid "Filter"
+msgstr "Lọc"
+
+#: templates/contest/ranking.html:584
+msgid "Clear"
+msgstr "Đặt lại"
+
```

Comment:
You don't need to include the localization in the PR, it should be done separately